handbook/product/design/design-process.md

```diff
@@ -121,15 +121,40 @@ While asynchronous communication is a core attribute of remote work, key moments
       - Schedule a meeting with engineers to discuss the interaction
   - Tools
       - Figma
-- Metrics collection:
+- **Metrics collection**
   - In the RFC process, the definition of success will include items which can be measured and evaluated
   - Involve the business operations team in determining what metrics to track and how they are tracked
   - Work with engineers to add the required eventLog tracking
   - Tools
       - Looker
       - Google analytics
-- Refinement
+- **Refinement**
   - Designers will review metrics and determine if their designs are meeting objectives set for them.
   - If areas for improvement are found, tickets will be created to document the change required to improve the issue
+  
+## Working with design files
+To provide consistency in design files handling, follow the rules listed below:
+
+#### File location
+All files should be created under the Sourcegraph organization in Figma. It is important to not work on Sourcegraph projects in the private files as designs should be accessible to all Team Members.
+
+#### Naming convention
+- All the files containing current project work must follow this naming structure:
+
+RFC or GitHub issue number + RCF or GitHub issue title + stage (Draft, Open to review, Final review or Approved).
+
+- Keep the stage updated as you progress with your work. 
```

Comment:
The formatting is a little confusing here since the non-bulleted items is technically a sub-bullet. What do you think of this suggestion?

```suggestion

All the files containing current project work must follow this naming structure:

> RFC or GitHub issue number + RFC or GitHub issue title + stage (Draft, Open to review, Final review or Approved).

Keep the stage updated as you progress with your work. 
```